### PR TITLE
startup: update to 2.0.3

### PR DESCRIPTION
**Pull request by CameronNemo**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/startup/template [hunk 1]
```diff
@@ -12,15 +12,20 @@ short_desc="Event driven task and service manager"
...
 	autoreconf -fi
 }
 
 post_install() {
+	local cmds="init-checkconf initctl2dot initctl reload restart start
+	 startup startup-monitor status stop"
+	for cmd in ${cmds}; do
+		vcompletion "contrib/fish_completion/${cmd}.fish" fish "$cmd"
+	done
 	rm -f -- "${DESTDIR}/usr/lib/libupstart.la"
```
Comment (ericonr) [inline srcpkgs/startup/template:29]: Please remove the `-f` flag.

File: srcpkgs/startup/template [hunk 2]
```diff
@@ -12,15 +12,20 @@ short_desc="Event driven task and service manager"
...
 
 pre_configure() {
 	autoreconf -fi
 }
 
 post_install() {
+	local cmds="init-checkconf initctl2dot initctl reload restart start
+	 startup startup-monitor status stop"
+	for cmd in ${cmds}; do
+		vcompletion "contrib/fish_completion/${cmd}.fish" fish "$cmd"
```
Comment (ericonr) [inline srcpkgs/startup/template:27]: Only fish completions?

File: srcpkgs/startup/template (hunk 1 above)
Comment (CameronNemo) [reply srcpkgs/startup/template]: done

File: srcpkgs/startup/template (hunk 2 above)
Comment (CameronNemo) [reply srcpkgs/startup/template:27]: yeah at least for now

**Pull request merged**